OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -82,6 +95,11 @@ public sealed class HackyAIInfo : IBotInfo, ITraitInfo
 		[Desc("Radius in cells around the center of the base to expand.")]
 		public readonly int MaxBaseRadius = 20;
 
+		[Desc("Radius in cells around each building with ProvideBuildableArea",
+		"to check for a 3x3 area of water where naval structures can be built.",
```

Comment:
While I agree this is bad (too RA-specific), IMO we should limit the scope creeping already.
Also as far as I see the code that uses this radius doesn't really care about the footprint, so I'd say just remove the size from the description here.